local/src/test/java/org/dockerImageTests/kafkaIT.java

```diff
@@ -0,0 +1,101 @@
+package org.dockerImageTests;

```

Comment:
All of the tests I see for the other images are written in Python, why did you write these in Java?

Reply:
Because we don't want to add python (something complete unrelated to running Kafka) on the docker image. Python would have taken lots of extra space while Java need to be present already on the image for running Kafka. So, we have tests written in Java.
I'm curious to know why the decision to include python was taken in all these other images at the first place.

Reply:
we were exploring testcontainers library for container lifecycle management of the image in tests, and this library has the most support in java.